cmd/crane/doc/crane_validate.md

```diff
@@ -0,0 +1,25 @@
+## crane validate
+
+Validate that an image is well-formed
+
+### Synopsis
+
+Validate that an image is well-formed
+
+```
+crane validate [flags]
+```
+
+### Options
+
+```
+      --daemon string    Name of image in daemon to validate
```

Comment:
How about `crane validate [image]` (positional arg) and if it matches a remote image ref we assume it's a remote image ref, otherwise check for a tarball file. `-L` can tell us to look for an image in the daemon, as we do elsewhere in `crane` and `ko`.

Reply:
I think there are some ambiguous cases and doing fallback means that we might match the wrong thing. I kind of like this pattern since it's easier to extend with new formats without having to thing about how that would interact with the others.